backend/src/services/api.rs

```diff
@@ -441,3 +441,101 @@ pub async fn raw_configuration_by_node(
         .await?,
     ))
 }
+
+/// Get Detailed Information on Node
+///
+/// # Errors
+///
+/// This function will return an error if the server was unable to get node'a client
+/// or one of the requests to get information from the node fails
+#[cfg_attr(feature = "swagger", utoipa::path(
+        get,
+        context_path = ApiV1::to_path(),
+        path = "/nodes/{node_name}/detailed",
+        params (
+            ("id", description = "Node's ID")
+        ),
+        responses(
+            (status = 200, body = DetailedNode, content_type = "application/json", description = "Detailed Node information"),
+            (status = 401, description = "Unauthorized"),
+            (status = 404, description = "Node Not Found")
+        ),
+        security(("api_key" = []))
+    ))]
+pub async fn get_detailed_node_info(
+    Extension(client): Extension<HttpBobClient>,
+    Path(node_name): Path<NodeName>,
+) -> AxumResult<Json<DetailedNode>> {
+    tracing::info!("get /nodes/{node_name}/detailed : {client:?}");
+    let handle = Arc::new(
+        client
+            .api_secondary(&node_name)
+            .cloned()
+            .ok_or(StatusCode::NOT_FOUND)?,
+    );
+
+    let status = {
```

Comment:
Probably this code can be extracted into macro, to prevent `handle...spawn` duplication?